server/server.go

```diff
@@ -412,14 +444,61 @@ func (s *Server) v0QueryPath(w http.ResponseWriter, r *http.Request, path ast.Re
 		return
 	}
 
+	if webhook {
+		var goInput interface{}
+		if input != nil {
+			if goInput, err = ast.ValueToInterface(input, stringResolver{}); err != nil {
```

Comment:
`ast.JSON` returns an error if it encounters a `Ref`. We just want to turn it into a string